notification-server/src/models/device.rs

```diff
@@ -5,7 +5,6 @@ use std::collections::HashSet;
 #[derive(Debug, Deserialize)]
 pub struct DeviceSubscription {
     pub preferences: Vec<Preference>,
-    // TODO: Investigate if this should be a AccountAddress instead of a String
     pub accounts:    Vec<String>,
```

Comment:
? 😆

Reply:
I was about to address and changed my mind while doing it

Reply:
```
    -H "Content-Type: application/json" \
    -d '{
        "preferences": [],
        "accounts": ["3CMdX2ypcD9JtKnwhpKXfQr233mJajF6hKMtHknn6MNxg"]
    }'
HTTP/2 400
date: Mon, 26 Aug 2024 13:23:40 GMT
content-type: application/json
content-length: 50

{"errorMessage":"Failed to parse account address"}
```

Reply:
Pretty nice status code as well. I am happy about it. :)

Reply:
```
HTTP/1.1 422 Unprocessable Entity
content-type: text/plain; charset=utf-8
content-length: 112
date: Mon, 26 Aug 2024 13:27:29 GMT

Failed to deserialize the JSON body into the target type: accounts[0]: Wrong Base58 version. at line 3 column 66%
```